lib/db.js

```diff
@@ -232,13 +224,13 @@ class LimitDBRedis extends EventEmitter {
   put(params, callback) {
     callback = callback || _.noop;
 
-    const valError = this.validateParams(params);
+    const valError = validateParams(params, this.buckets);
```

Comment:
Required to ensure the parameter key matches a bucket type, formerly accessible via `this`, but no longer once it's been externalized.